packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -24,6 +24,11 @@ export const DEFAULT_LOCALE_TEXT: LocaleText = {
   toolbarFiltersTooltipShow: 'Show Filters',
   toolbarFiltersTooltipActive: (count) => `${count} active filter(s)`,
 
+  // Export selector toolbar button text
+  toolbarExport: 'Export',
+  toolbarExportLabel: 'Export',
+  toolbarExportCSV: 'CSV',
```

Comment:
I think that more context on the action would help:

```suggestion
  toolbarExportCSV: 'Download as CSV',
```